plasmapy/atomic/tests/test_particle_class.py

```diff
@@ -2,6 +2,7 @@
 import numpy as np
 from astropy import units as u
 import inspect
+import roman
 
 from ...constants import m_p, m_e, m_n, e
 
```

Comment:
Add some tests for the new `roman_symbol` method?

Reply:
According to Coveralls it looks like the tests that @Elfhelm added below cover everything except the `roman.OutOfRangeError` line, which I'm not too worried about.  Might not be a bad idea to sooner or later add a `Particle` class test of a negatively charged ion, like "H 1-" (which is important for opacity in stellar atmospheres for relatively cool stars, surprisingly enough).

Reply:
Fair enough!